assets/src/edit-story/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -120,21 +129,48 @@ const getVideoResource = async (file) => {
   });
 };
 
+const createPlaceholderResource = (properties) => {
+  return createLocalResource({ ...properties, isPlaceholder: true });
+};
+
+const getPlaceholderResource = (file) => {
+  const type = getTypeFromMime(file.type);
+  const mimeType = type === 'image' ? 'image/png' : 'video/mp4';
```

Comment:
Why can't we use the real mimeType here?

Reply:
`file.type` can be empty, or something like `video/quicktime` which the browser won't support.

The exact mime type used here is irrelevant as it is merely a placeholder without a `src`, not a real media item.